src/bgw/scheduler.c

```diff
@@ -216,6 +227,7 @@ scheduled_bgw_job_transition_state_to(ScheduledBgwJob *sjob, JobState new_state)
 	switch (new_state)
 	{
 		case JOB_STATE_DISABLED:
+			/* we don't have a way of getting to this state now */
```

Comment:
This comment is a bit confusing given the transition to this state on line 243. If this is indeed true, maybe the transition on line 243 should be removed.